src/pages/Home/Home.js

```diff
@@ -1,11 +1,51 @@
-import React from 'react';
+import React from "react";
 
-import './Home.scss';
+import "./Home.scss";
+import MediumCardBlogPost from "components/PostMediumBlogPost";
+import JoinMadhyamCard from "components/JoinMadhyamCard";
+import SmallCardBlogPost from "components/SmallCardBlogPost";
+import PostsTrendingCard from "components/PostsTrendingCard";
+import BlogPostCard from "components/BlogPostCard";
 
-export default props => {
-    return(
-        <div className="HomePage">
-            React Starter Pack for ITSNP
-        </div>
-    );
-}
\ No newline at end of file
+import MOMO from "../../resources/images/global/momo.png";
+import SALAD from "../../resources/images/global/salad.png";
+import ITSNP from "../../resources/images/global/itsnp.png";
+import USER from "../../resources/images/global/user.png";
+import SANDWICH from "../../resources/images/global/sandwich.png";
+
+export default (props) => {
+  return (
+    <div className="homePage">
+      <PostsTrendingCard
+        image={MOMO}
+        caption="momo"
```

Comment:
Why is the image caption name `caption `here and elsewhere `imagetitle`?
Please maintain consistency in naming

Reply:
changed the caption  to imageTitle in all component